docs/bonus-labs/avi-ldap-auth/avi-ldap-auth.md

```diff
@@ -0,0 +1,169 @@
+# Configure NSX ALB Auth with Okta LDAP
+
+The standard [NSX ALB Lab](../avi/setup_avi_ctrl.md) leverages as single local NSX ALB admin account.  However, the TKG labs leverage Okta for enterprise authentication (via OIDC).
+
+NSX ALB has the ability to configure authentication policies to integrate with enterprise IdPs.  However, it does not support OIDC.  Good news is that Okta provides for an LDAP interface to the same member repository that you setup for the TKG labs.
+
+In this bonus lab, we will configure the LDAP Interface for Okta and then configure NSX ALB so that members of the `platform-team` group can perform administrative functions on NSX ALB.
+
+## Configure Okta for LDAP Integration
+
+- Setup the configuration
```

Comment:
These 3 bulletpoints don't match with the 3 sub-sections bellow in name

Reply:
Thanks.  Fixed.